src/karmabot/settings.py

```diff
@@ -45,5 +45,5 @@ def _get_env_var(env_var: str):
 # Karma
 # the first +/- is merely signaling, start counting (regex capture)
 # from second +/- onwards, so bob++ adds 1 point, bob+++ = +2, etc
-KARMA_ACTION = re.compile(r"(?:^| )(\S{2,}?)\s?[\+\-]([\+\-]+)")
+KARMA_ACTION = re.compile(r"(?:^| )(\S{2,}?)\s?[\+\-]([\+\-]+)", flags=re.MULTILINE)
```

Comment:
That was easy haha

Reply:
sweet, glad you added a test as I sometimes run into issues with `re.M`, let me push and we test it out